content/advent-2017/go-grpc-beyond-basics.md

```diff
@@ -0,0 +1,501 @@
++++
+author = ["Pieter Louw"]
+date = "2017-12-13"
+series = ["Advent 2017"]
+linktitle = "gRPC Go: Beyond the basics"
+title = "gRPC Go: Beyond the basics"
+
```

Comment:
I don't know whether it affects parsing or not, but you should remove this blank line (unless you have a good reason for including it). Other posts don't have it. See https://raw.githubusercontent.com/gopheracademy/gopheracademy-web/master/content/advent-2017/mphf.md for example.